docs/team/tlylt.md

```diff
@@ -3,66 +3,71 @@ layout: page
 
-- **New Feature**: Added the ability to undo/redo previous commands.
+#### New Features & Enhancements
 
-  - What it does: allows the user to undo all previous commands one at a time.
-    Preceding undo commands can be reversed by using the redo command.
+- **New Feature**: Added the ability to send a request without saving the endpoint.
+
+  - What it does: allows the user to run API requests on the fly. 
   - Justification: This feature improves the product significantly because a
-    user can make mistakes in commands and the app should provide a convenient
-    way to rectify them.
-  - Highlights: This enhancement affects existing commands and commands to be
-    added in future. It required an in-depth analysis of design alternatives.
+    user can quickly type in an API URL with specifying details, and the app 
+    will conveniently trigger a request for the user to view the response.
+  - Highlights: This enhancement affects existing commands that triggers an outbound
+    request. It required an in-depth analysis of design alternatives.
     The implementation too was challenging as it required changes to existing
-    commands.
-  - Credits: _{mention here if you reused any code/ideas from elsewhere or if a
-    third-party library is heavily used in the feature so that a reader can make
-    a more accurate judgement of how much effort went into the feature}_
+    commands and how we verify the validity of URLs.
+
+- **New Feature**: Added a keyboard shortcut that allows the user to retrieve 
+  the previous valid command using ctrl + up-arrow (Windows) / cmd + up-arrow (Mac).
 
-- **New Feature**: Added a history command that allows the user to navigate to
-  previous commands using up/down keys.
+- **New Feature**: Added a help command that allows the user to open a pop up window displaying
+   relevant links such as the product's user guide and also a table of command summary. 
 
-- **Code contributed**: [RepoSense link]()
+- **New Feature**: Added a show command that allows the user to view the details of a saved endpoint.
 
 - **Project management**:
 
-  - Managed releases `v1.3` - `v1.5rc` (3 releases) on GitHub
+  - Managed releases `v1.4` on GitHub
 
 - **Enhancements to existing features**:
 
-  - Updated the GUI color scheme (Pull requests [\#33](), [\#34]())
-  - Wrote additional tests for existing features to increase coverage from 88%
-    to 92% (Pull requests [\#36](), [\#38]())
+  - Updated various custom feedback messages in the result display for commands
```

Comment:
Nit picking at this point but some bullet points have full stop and some don't :3

Reply:
Thank you, resolved in my updated commit